kubernetes/cmsweb/scripts/fix_highdns.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
```

Comment:
Please add here a doc string describing use-case, e.g. message we get from network team, etc.